public/app/features/dashboard/components/DashNav/DashNav.tsx

```diff
@@ -338,6 +339,18 @@ export const DashNav = React.memo<Props>((props) => {
 
     buttons.push(renderTimeControls());
     buttons.push(tvButton);
+
+    if (config.featureToggles.scenes) {
+      buttons.push(
+        <Button
+          size="sm"
+          variant="secondary"
+          onClick={() => locationService.push(`/scenes/dashboard/${dashboard.uid}`)}
+        >
+          View as Scene
+        </Button>
```

Comment:
Can we make this a ToolbarButton with just an icon? (like apps) or just a single letter, this will be quite distracting for many months (same on the Scene side). 

Or just a keyboard shortcut :) 

I take screenshots a lot and this will just mess them up.

Reply:
but it's a great idea, the experience of switching between the two is really cool and you see right away differences!